etc/adminhtml/menu.xml

```diff
@@ -81,5 +81,14 @@
             action="wizzy_search/sync/skippedentities"
             resource="Wizzy_Search::main"
         />
+         <add
```

Comment:
Remove this from Menu please.

Reply:
Can i put it under 'Advanced Configuration' ?